### DISCUSSION TITLE
[@types/cytoscape-edgehandles] Wrong parameter of EdgeHandlesInstance.start()

### DISCUSSION THREAD

**Discussion by porst17** [DefinitelyTyped/DefinitelyTyped#60279] [Issues with a @types package]
The signature exposed for the `EdgeHandlesInstance.start()` method is currently `start: (sourceNode: string) => void`. However, `start()` does not except a string, but rather a node instance (with methods like `.id()` etc.). I suppose, the signature should be changed to `start: (sourceNode: cytoscape.NodeSingular) => void`.

[a comment by an automated account is not shown]

**Comment by porst17** [comment]
Is there any chance of getting this fixed soon?

**Comment by jzizka8** [comment]
this issue still persists

**Comment by ghost** [comment]
Created PR #69154 to fix this.
